packages/client/src/generation/TSClient/Model.ts

```diff
@@ -279,7 +279,43 @@ export type ${getAggregateGetName(model.name)}<T extends ${getAggregateArgsName(
     const modelLine = `Model ${model.name}\n`
     const docs = `${modelLine}${docLines}`
 
-    return `${buildComment(docs)}export type ${model.name} = {
+    return ifExtensions(
+      () => {
+        return `export type ${
+          model.name
+        }Payload<ExtArgs extends runtime.Types.Extensions.Args = runtime.Types.Extensions.DefaultArgs> = {
+${indent(`kind: 'Payload'`, TAB_SIZE)}
```

Comment:
What function does `kind` field have?

Reply:
I was used to discriminate but isn't useful anymore. Will remove.